src/App.css

```diff
@@ -0,0 +1,7 @@
+h1 {
+  font-size: 2.25rem;
```

Comment:
Instead of doing this, try using MUI’s Typography so we don’t need to guess which fontstyles we need to use. We can also adjust the style later on the component.

Reply:
Okayy, noted!!